### PR TITLE
PR: Assign a keyboard shortcut for "next figure" and "previous figure" of the Plots pane

### LINKED ISSUE #8641: Assign a keyboard shortcut for "next figure" and "previous figure" of the Plots pane
I think it would be nice to add shortcuts for "next figure", "previous figure" of the Plots pane.
Thank you.

### PR DESCRIPTION
**Pull request by ok97465**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)
* [X] Included a screenshot (if affecting the UI)

### Issue(s) Resolved
figure_shortcut

Fixes #8641 

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: ok97465

### CODE REVIEW

**Comment by ok97465** [comment]
I read the test log but I don't know what caused the test to fail.

**Comment by Khalilsqu** [comment]
> Thank you @Khalilsqu. I think I should study more about git.

i am not good at it neither, :grin:

**Comment by ccordoba12** [comment]
Thanks @ok97465 for your contribution.

@jnsebgosselin, could you take a look at this one? Thanks!

**Comment by jnsebgosselin** [comment]
Sure. I will do it tonight.

File: spyder/plugins/plots/widgets/figurebrowser.py [hunk 1]
```diff
@@ -168,12 +168,14 @@ def setup_toolbar(self):
 
         goback_btn = create_toolbutton(
                 self, icon=ima.icon('ArrowBack'),
-                tip=_("Previous Figure"),
+                tip=_("Previous Figure (%s)" %
```
Comment (jnsebgosselin) [inline spyder/plugins/plots/widgets/figurebrowser.py]: Please use the new [format](https://docs.python.org/3.4/library/functions.html#format) method for str. I think this is what we try to use now whenever we can. So what I mean is to do this instead:
```python
tip=_("Previous Figure ({})".format(
    get_shortcut('plots', 'previous figure')))
```

File: spyder/plugins/plots/widgets/figurebrowser.py [hunk 2]
```diff
@@ -168,12 +168,14 @@ def setup_toolbar(self):
...
+                      get_shortcut('plots', 'previous figure')),
                 triggered=self.go_previous_thumbnail)
 
         gonext_btn = create_toolbutton(
                 self, icon=ima.icon('ArrowForward'),
-                tip=_("Next Figure"),
+                tip=_("Next Figure (%s)" %
+                      get_shortcut('plots', 'next figure')),
```
Comment (jnsebgosselin) [inline spyder/plugins/plots/widgets/figurebrowser.py]: Same comment as above.

**Comment by jnsebgosselin** [comment]
I left some very minor comment. Otherwise, everything looks good and I also tested it locally. Thanks @ok97465 .

**Pull request merged**